projects/index.php

```diff
@@ -231,6 +231,7 @@ function isNumeric(sText){
 
 	<!-- This is inner text! -->
 	<div id="innertext">
+		<h1 class="page-heading"><?= $LANG['INVPROJ']; ?></h1>
```

Comment:
Make screenreader only and switch to "View Inventory Project"

Reply:
Fixed in 3a0cbc022.